### DISCUSSION TITLE
Welcome to openpracticelibrary Discussions!

### DISCUSSION THREAD

**Discussion by kattekrab** [openpracticelibrary/openpracticelibrary#2063] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kattekrab** [comment]
Is GitHub a good place to gather?

**Reply by tenfourty** [reply]
I would definitely prefer a persistent forum like experience over a chat like experience (slack or discord).

**Reply by kattekrab** [reply]
Thanks @tenfourty.  This is great feedback. It's a really good point.  A persistent forum is much more easily referenced, and is easier to engage with asynchronously.  And for a global community like ours is, that's important.

Would you be willing to "think out loud" a bit on why you prefer forum to chat?

When I interviewed @webchick about the OPL, she told me she went looking for some form of real time chat to ask real time questions, like IRC, or Slack. When I spoke to @pendashteh he talked about the different kinds of communities that grow around platforms like mediawiki, vs drupal.

I expect people will have different requirements at different times and we may want to think about balancing those needs against fracturing our channels. Our community communication style will be informed by the kind of platforms we choose.  I'd like to be intentional about this as we set up enabling structures. (Thanks to @shabnoorshah for that phrase! :)

**Reply by pendashteh** [reply]
I was thinking ah what a great term where I saw the credit is followed immediately!
Enabling Structures 
Related to this, my favourite term is Enabling Environments.

**Reply by tenfourty** [reply]
I've been thinking a bit about this.

GitHub is pretty good in that it is open and free, however, the experience is very technical and certainly, the destination is better known to folks in tech.

Slack is very well known, many folks have it for work so they can switch between different Slack workspaces. I just wonder about using a tool that is optimised for chat rather than for long-form discussion (also there is a messaging limit and it's hard to get members to pay for Slack so you are often stuck with a free instance). I still see why real-time chat could be important for some folks.

One of the words we are using a lot here is "community" and I wonder if it might make sense to look at a more community-oriented platform like [circle.so](https://circle.so/) (it is a lot more expensive than other options so I'm not sure it's a great fit for us but the community aspect is interesting).

Beyond pure forum/chat style tools (like Slack/Discord/Discourse etc) there could be other options like hosting our own reddit or stack overflow site or even our own mastodon server (fediverse twitter).

I agree that figuring out what we want to enable first (I love the terms enabling structures/environments too!) is step one on this journey.

**Comment by kattekrab** [comment]
Thanks for sharing your thoughts here Jeremy. Yep that pretty much aligns with my thinking too. 

The idea of running our own mastodon server is kind of growing on me to be honest. I’ve not heard of circle.so before. I’ll check it out. 

But first, yes, we need to determine “what do we want to achieve?”Objective. “How do we know when we‘ve reached that goal?” Metric. Before investing in tooling. I wonder if some experiments might be in order?

Via mobile.

**Comment by kattekrab** [comment]
Follow up - hilariously... I am actually a member of a circle.so community! And hadn't realised.  I've not been super active there though, so have not formed any useful opinions.

For now, I'm leaning towards sticking with github, and linkedin. 
We also have a forum that comes with opencollective, which is currently only being used to update our financial contributors.

I do think it might be fun to experiment with a mastodon instance, but as I've barely got enough time and people to manage the site itself, I don't want to add more work to the backlog at the moment.

We do still have a slack instance, but nothing happens there, other than notifications from github actions about deployments.
If any of you are interested in joining it, please leave a note here, and I'll sort out invitations.

cheers
Donna
